### PR TITLE
[ACCESSIBILITY 4.1.2] Homepage - Carousel dots innerText

### PR DESCRIPTION
**Pull request by tarantilis**
https://taskman.eionet.europa.eu/issues/254326

https://62713c57bcbdf4004a3d99b8-sphuwfnnjq.chromatic.com/?path=/story/components-card-default--carousel-cards

### CODE REVIEW

**Comment by tarantilis** [comment]
@eea-jenkins build storybook

File: src/ui/Card/Card.stories.jsx [hunk 1]
```diff
@@ -455,6 +455,17 @@ function CarouselCardsContent({
   cards,
 }) {
   const slider = React.useRef(null);
+
+  useEffect(() => {
```
Comment (tarantilis) [inline src/ui/Card/Card.stories.jsx:459]: Please check if you can recreate this implementation in volto.
Could not find any other way to refactor the dots except useEffect.

Comment (ichim-david) [reply src/ui/Card/Card.stories.jsx:459]: The implementation is actually easy without the need for an useEffect see
the change I made directly in develop https://github.com/eea/volto-eea-design-system/commit/fe5eb46dc94abfeaaa2c79022e507b452b4bb36a

**Comment by ichim-david** [comment]
@tarantilis  I will not merge these changes as card.stories.jsx was already heavily refactored by me and I found a better solution for customizing the dots button text as seen in the comments I gave as such I will close this pull request

**Pull request closed without merging**